packages/ruleset/src/functions/parameter-description.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function(param, _opts, { path }) {
+  return parameterDescription(param, path);
+};
+
+const errorMsg = 'Parameter should have a non-empty description';
+
+function parameterDescription(param, path) {
+  const results = [];
```

Comment:
I was reading through the Spectral custom function guide the other day and noticed they recommend against defining too many variables for performance reasons, since these functions may be called lots and lots of times.

For many of our functions, this is fairly unavoidable and it's fine. For this, since we aren't gathering multiple errors per function call, we could probably optimize a bit by returning an inline-defined array if the condition is met and returning an empty array at the end of the function.

This is just a suggestion though, not necessary. I'll leave it up to you

Reply:
done